pheno005/phenotype/demographics.json

```diff
@@ -0,0 +1,75 @@
+{
+	"participant_id": {
+		"Description": "OpenNeuro ID of the subject."
+	},
+	"session_id": {
+		"Description": "ID assigned to participant's visit.",
+		"Levels": {
+			"baseline": "First imaging and phenotype data collection visit.",
+			"followup": "Second phenotype data collection visit."
+		}
+	},
+	"run_id": {
+		"Description": "ID assigned to a run for a given session.",
+		"Levels": {
+			"01": "First run of the session.",
+			"02": "Second run of the session."
+		}
+	},
+	"age": {
+		"Description": "Age of the participant.",
+		"Units": "years"
+	},
+	"sex": {
```

Comment:
I'm conflicted on whether sex at birth should go in the participants.tsv or the demographics.tsv. I know OpenNeuro crawls participants.tsv files better right now to improve its search functionality, which may be justification enough to move this. Anyway, this is a point of discussion and I am open to other ideas.

Reply:
@ericearl Based on our slack discussion, I think we can leave the demographics info in the demographics file with participants.tsv being a list of unique participant IDs?